### PR TITLE
Essential notifications script

### PR DESCRIPTION
**Pull request by olegnn**
Basic implementation of the script for essential notifications.

### CODE REVIEW

File: scripts/watch_txs.js [hunk 1]
```diff
@@ -0,0 +1,783 @@
...
+    democracyTx("second"),
+    ({
+      tx: {
+        args: [idx],
+      },
+    }) => of(`Proposal #${idx.toString()} is seconded`)
+  ),
+
+  // `force_transfer` call
+  checkMap(balancesTx("forceTransfer"), ({ tx }) => {
```
Comment (lovesh) [inline scripts/watch_txs.js:419]: Should also check for `setBalance`

File: scripts/watch_txs.js [hunk 2]
```diff
@@ -0,0 +1,783 @@
...
+              return of(acc);
+            }
+          }),
+          mapRx((acc) => {
+            const newDiff = acc.handle(newMembers.toJSON().map(head));
+            councilDiffAccumulator.next(newDiff);
+
+            return newDiff;
+          }),
+          pluck("diff"),
```
Comment (lovesh) [inline scripts/watch_txs.js:557]: `pluck` is deprecated btw https://rxjs.dev/api/index/function/pluck

File: scripts/watch_txs.js [hunk 3]
```diff
@@ -0,0 +1,783 @@
...
+      both(democracyTx("fastTrack"), democracyEvent("Started")),
+      ({
+        tx: {
+          args: [proposalHash],
+        },
+      }) => of(`Proposal is fast-tracked: ${proposalHash.toString()}`)
+    ),
+
+    // Council proposal
+    checkMap(councilEvent("Proposed"), ({ event }, dock) => {
```
Comment (lovesh) [inline scripts/watch_txs.js:494]: Event for closing proposal?

**Review by lovesh — APPROVED** [review]

File: scripts/watch_txs.js (hunk 3 above)
Comment (olegnn) [reply scripts/watch_txs.js:494]: https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR431

File: scripts/watch_txs.js (hunk 1 above)
Comment (olegnn) [reply scripts/watch_txs.js:419]: https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR569

Comment (lovesh) [reply scripts/watch_txs.js:436]: Did you mean to redirect to the `BalanceSet` event?

File: scripts/watch_txs.js (hunk 3 above)
Comment (lovesh) [reply scripts/watch_txs.js:511]: Its redirecting me to `checkMap(democracyTx("second"),`, is that where you intended?

File: scripts/watch_txs.js (hunk 1 above)
Comment (olegnn) [reply scripts/watch_txs.js:436]: Right, just something wrong with the link
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR586

File: scripts/watch_txs.js (hunk 3 above)
Comment (olegnn) [reply scripts/watch_txs.js:511]: Some offset was added with new commits, my apologies
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR448

File: scripts/watch_txs.js (hunk 1 above)
Comment (lovesh) [reply scripts/watch_txs.js:436]: Thanks.

File: scripts/watch_txs.js (hunk 3 above)
Comment (lovesh) [reply scripts/watch_txs.js:511]: Thanks.

**Pull request merged**